### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Category/ModuleCat/Presheaf/Sheafify.lean [hunk 1]
```diff
@@ -320,11 +320,11 @@ def toSheafify : M₀ ⟶ (restrictScalars α).obj (sheafify α φ).val where
     simpa using (Sheafify.map_smul_eq α φ (α.app _ r₀) (φ.app _ m₀) (𝟙 _)
       r₀ (by aesop) m₀ (by simp)).symm
 
-instance : Presheaf.IsLocallyInjective J (toSheafify α φ).hom :=
-  by dsimp; infer_instance
+instance : Presheaf.IsLocallyInjective J (toSheafify α φ).hom := by
```
Comment (grunweg) [inline Mathlib/Algebra/Category/ModuleCat/Presheaf/Sheafify.lean:323]: If you prefer to automate such changes: #13420 has a style linter (with auto-fix) for this - and is looking for review.

File: Mathlib/FieldTheory/Adjoin.lean [hunk 2]
```diff
@@ -1396,8 +1396,8 @@ open AlgEquiv
 variable {K L : Type _} [Field K] [Field L] [Algebra K L]
 namespace AdjoinRoot
 
-/-- The canonical algebraic equivalence between `AdjoinRoot p` and `AdjoinRoot q`, where
-  the two polynomial `p q : K[X]` are equal.-/
+/-- The canonical algebraic homomorphism from `AdjoinRoot p` to `AdjoinRoot q`, where
```
Comment (grunweg) [inline Mathlib/FieldTheory/Adjoin.lean:1399]: Nice find!

File: Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean [hunk 3]
```diff
@@ -311,10 +311,12 @@ variable [NumberField K]
 
 theorem nnnorm_eq_sup_normAtPlace (x : E K) :
     ‖x‖₊ = univ.sup fun w ↦ ⟨normAtPlace w x, normAtPlace_nonneg w x⟩ := by
-  rw [show (univ : Finset (InfinitePlace K)) = (univ.image
-    (fun w : {w : InfinitePlace K // IsReal w} ↦ w.1)) ∪
-    (univ.image (fun w : {w : InfinitePlace K // IsComplex w} ↦ w.1))
-    by ext; simp [isReal_or_isComplex], sup_union, univ.sup_image, univ.sup_image, sup_eq_max,
+  have :
```
Comment (grunweg) [inline Mathlib/NumberTheory/NumberField/CanonicalEmbedding/Basic.lean:314]: Much nicer, thanks!

File: Mathlib/Topology/Algebra/InfiniteSum/Order.lean [hunk 4]
```diff
@@ -83,11 +83,13 @@ theorem tprod_le_tprod_of_inj {g : κ → α} (e : ι → κ) (he : Injective e)
 
 @[to_additive]
 lemma tprod_subtype_le {κ γ : Type*} [OrderedCommGroup γ] [UniformSpace γ] [UniformGroup γ]
-    [OrderClosedTopology γ] [ CompleteSpace γ] (f : κ → γ) (β : Set κ) (h : ∀ a : κ, 1 ≤ f a)
+    [OrderClosedTopology γ] [CompleteSpace γ] (f : κ → γ) (β : Set κ) (h : ∀ a : κ, 1 ≤ f a)
     (hf : Multipliable f) : (∏' (b : β), f b) ≤ (∏' (a : κ), f a) := by
-  apply tprod_le_tprod_of_inj _ (Subtype.coe_injective) (by simp only [Subtype.range_coe_subtype,
-    Set.setOf_mem_eq, h, implies_true]) (by simp only [le_refl,
-    Subtype.forall, implies_true]) (by apply hf.subtype)
+  apply tprod_le_tprod_of_inj _
```
Comment (grunweg) [inline Mathlib/Topology/Algebra/InfiniteSum/Order.lean:88]: Much nicer, thanks!

**Review by grunweg — COMMENTED** [review]
Thanks for doing this - I like all the changes.
maintainer merge

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**